folding-schemes/src/folding/nova/decider_eth_circuit.rs

```diff
@@ -0,0 +1,686 @@
+/// This file implements the onchain (Ethereum's EVM) decider circuit. For non-ethereum use cases,
```

Comment:
this file appears as 'new' when it should be as 'file renamed without changes' (since it exist in the `main` branch: https://github.com/privacy-scaling-explorations/folding-schemes/blob/main/src/folding/nova/decider_eth_circuit.rs )
Could it be that the PR branch is not getting all the changes from the `main` branch?

Reply:
Also for `pedersen.rs`. I've done in this branch an update getting last `main` version into `folding-schemes` subdir, if that is useful: https://github.com/privacy-scaling-explorations/folding-schemes/compare/main...feature/solidity-verifiers-from-last-main

Reply:
Yes, I will close this PR and move to your branch. 

I've done a rebase early on, but on a branch different from main, to get your kzg implementation when testing. I think that's why we might end up with this! Sorry for the trouble :)